README.md

```diff
@@ -92,6 +107,9 @@ assertNotDisplayed("Hello world");
 assertNotDisplayed(R.string.hello_world);
 assertNotDisplayed(R.id.button);
 
+// Is this text displayed in this exact view?
+assertDisplayed(R.id.view, "Hello world")
```

Comment:
Maybe move it next to the other `assertDisplyed`?

Reply:
Hmmm that kinda breaks the `// blahblahblah | // ... or not` flow.
But my guts also tell me that you're right.

I'll do it anyways 🙈

Reply:
I know what you mean....

Maybe adding an `assertNotDisplayed(R.id.view, "Hello world")`? But it might not be really useful...

On the other hand, it's kinda strange to have two `assertDisplayed`separated, if they share the method name, they must do "the same"... Are we facing a naming problem (again)?

I don't know 💃

Reply:
A savage issue appeared! https://github.com/SchibstedSpain/Barista/issues/164

Reply:
Leave it to me!

![gotta_catch_ em_all](https://user-images.githubusercontent.com/6932449/32842119-8d0bcb94-ca1c-11e7-842b-c1c719eba5ba.png)

Reply:
Catch! #165